app/helpers/dropdown_helper.rb

```diff
@@ -20,30 +20,38 @@ def audit_status_options
   end
 
   def taxons_options
-    options = @search.options_for("taxons").order(:title)
+    options = ContentItem
+      .where(document_type: "taxon")
+      .order(:title)
 
     options_from_collection_for_select(
       options,
       :content_id,
-      :title_with_count,
+      :title,
       selected: params["taxons"],
     )
   end
 
   def organisation_options
-    options = @search.options_for(org_link_type).order(:title)
+    options = ContentItem
+      .where(document_type: "organisation")
+      .order(:title)
 
     options_from_collection_for_select(
       options,
       :content_id,
-      :title_with_count,
+      :title,
       selected: params[:organisations],
     )
   end
 
   def document_type_options
-    options = @search.options_for(:document_type)
-    options = options.map { |o| DocumentTypeOption.new(o) }
+    options = ContentItem
+      .all
+      .select(:document_type)
+      .group(:document_type)
+      .sort_by(&:document_type)
+      .map { |option| DocumentTypeOption.new(option) }
```

Comment:
I might be missing something, but you are returning here a non-unique list of document types. It would be quicker to do something like:

```ruby
ContentItem.pluck(:document_type)
```

And, I would say that you need here the unique list, so what about doing this:

```ruby
ContentItem.pluck(:document_type).uniq
```